src/improvements/tasks/eth/001-opcm-upgrade-v200/config.toml

```diff
@@ -0,0 +1,26 @@
+l2chains = [
+    {name = "Base", chainId = 8453}
+]
+
+templateName = "OPCMUpgradeV200"
+
+[opcmUpgrades]
+absolutePrestates = [
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 8453},
```

Comment:
This isn't the correct absolute prestate, should be `0x0354eee87a1775d96afee8977ef6d5d6bd3612b256170952a01bf1051610ee01`

Reply:
Thanks